include/common/logger.h

```diff
@@ -1 +1,55 @@
-class logger{};
\ No newline at end of file
+#pragma once
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <chrono> 
+
+class Logger {
+public:
+	enum loglevel
+	{
+		logCRITICAL, logERROR, logWARNING, logDEBUG, logINFO,
+	};
+	Logger(std::string file_name, loglevel level) : full_buffer_size (100)
+	{
+		log_level_needed = level;
+		out.open(file_name, std::ofstream::out | std::ofstream::trunc);
+	}
+	~Logger()
+	{
+		out << full_buffer;
+		out.close();
+	}
+	const char* timenow()
+	{
+		auto timenow = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+		return  ctime(&timenow);
+	}
+
+	template<typename... Ts>
+	void hlog(int log_level, std::string word, Ts ... args)
```

Comment:
why word? is it a single word? maybe line should be better

Reply:
renamed to line